lib/howl/application.moon

```diff
@@ -313,6 +313,12 @@ class Application extends PropertyObject
     while count < max_count and C.g_main_context_iteration(ctx, false) != 0
       count += 1
 
+  _append_buffer: (buffer) =>
+    if config.autoclose_single_buffer and #@_buffers == 1 and not @_buffers[1].file and not @_buffers[1].modified
```

Comment:
This can be stricter. While unlikely, it's possible to have an unmodified buffer with no file that would still be interesting. I suggest requiring a couple of different criteria for the buffer:

- No associated file
- Not modified
- Empty
- Optionally we could also check that there haven't been any revisions added